docs/adrs/004-SplitAndAggregate.md

```diff
@@ -125,12 +125,21 @@ Still to be determined (not in scope of this ADR):
 * `+` Simplest solution, quickest turnaround, fixes deadlock issue
 * `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs 
 
-### Increase number of slots or Seperate Queue for kbparallels apps without 10 job limit
-* `+` Simple solutions, quick turnarounds, fixes deadlock issue
+### Increase number of slots per user > 10
+* `+` Simple solutions, quick turnarounds, fixes deadlock issue for small numbers of jobs.
+* `-` Doesn't fix deadlock issue as the user can still submit more KBP jobs
 * `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs
 * `-` A small amount of users can take over the entire system
 * `-` The calculations done by the apps will interfere with other apps and cause crashes/failures
 
+###  Seperate Queue for kbparallels apps that may or may not have its own limit to running jobs.
+* `+` Simple solutions, quick turnarounds, fixes deadlock issue 
+* `+` Requires minimum changes to ee2 and condor if condor supports this feature
+* `-` Addresses only the deadlocking issue, UI still broken for regular runs and batch runs
+* `-` A small amount of users can take over the entire system unless  the new queue has its own limit to running jobs then it prevents users from taking over.
```

Comment:
I don't think this is true, is it? They could dominate the KBP queue if there's no limit but the other queues should be immune. I think the answer here is there needs to be a job count limit on the KBP queue as well. Ideally we could have multiple jobs per processor as they're supposed to be light weight

Reply:
Changed.
We are not working in the ideal situation, because right now the kbp jobs do computation as well. That would require rewriting all apps that use kbparallels that are doing computation which is not in the scope of this solution.